components/kubernetes/kind-cluster.yaml

```diff
@@ -1,10 +1,12 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
-- role: control-plane
-  extraMounts:
-  - hostPath: /proc
-    containerPath: /host/proc
+  - role: control-plane
+    extraMounts:
+      - hostPath: /proc
+        containerPath: /host/proc
```

Comment:
I will put again my comment from https://github.com/DataDog/test-infra-definitions/pull/774#discussion_r1570246532

With Kind we're not testing with real host proc or host sys as we are not interested in the PIDs and containers not managed by Kind, we never had any issue so far, so I am not sure the use case is valid.

Reply:
I see, I'm not 100% sure if container monitoring will be impacted or not. If not that's fine and can be merged, although it should be explained around the extra mounts.